juniper_codegen/Cargo.toml

```diff
@@ -19,7 +19,7 @@ proc-macro = true
 
 [dependencies]
 proc-macro-error = "1.0.2"
-proc-macro2 = "1.0.1"
+proc-macro2 = "1.0.2"
```

Comment:
Cargo by default uses [carret version range](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements), so the specified version is not the exact version being picked up during deps resolution, but rather a most bottom possible one.

That's why there is no reason to bump `proc-macro2` to higher version here, unless:
- we want to use in the library code a feature available only in a newer version of dependency;
- we want to upgrade to a new major version of dependency, which is not covered with this carret version range, i.e. outside `>=1.0.2, <2.0.0`;
- a current the most bottom version of dependency is [yanked](https://doc.rust-lang.org/cargo/commands/cargo-yank.html).